.travis.yml

```diff
@@ -3,16 +3,14 @@ sudo: required
 
 dart:
   - 1.24.3
-  - stable
```

Comment:
Should we leave stable for once 2.0 is released?

Reply:
I'd like it to run on dart 2 today ... so maybe we should just specify 2.0.0-dev.69.5 and then update it to 2.0.0 / stable once 2.0 lands?

Reply:
Yeah I was just thinking that once Dart 2 stable lands, we'd come back and update these to:
```yaml
dart:
  - 1.24.3
  - stable
  - dev
```

Reply:
Or maybe we leave the `dev` channel out at that point.

Reply:
I think we should do all three